lib/io_handler.rb

```diff
@@ -65,16 +65,14 @@ def self.get_user_input
     $stdin.gets.chomp
   end
 
-  def self.display_game_over_msg(move_number, player_num, playing_computer)
-    if move_number >= 9
+  def self.display_game_over_msg(player_won, player_num, playing_computer)
```

Comment:
There are a lot of conditionals in this function.  Good OO code contains very few if statements.  How could you refactor to not need so many conditionals?